e2e/content-services/upload/upload-dialog.e2e.ts

```diff
@@ -231,18 +232,23 @@ describe('Upload component', () => {
         await uploadDialog.dialogIsNotDisplayed();
 
         await uploadToggles.enableFolderUpload();
+
         await browser.executeScript(` setInterval(() => {
                if(document.querySelector('[data-automation-id="adf"]')){
                     document.querySelector("#adf-upload-dialog-cancel-all").click();
                     document.querySelector("#adf-upload-dialog-cancel").click();
                 }
-              }, 500)`);
+              }, 2000)`);
         await contentServicesPage.uploadFolder(adfBigFolder.location);
 
-        await expect(await uploadDialog.getTitleText()).toEqual('Upload canceled');
+        await uploadDialog.fileIsUploaded('a_png_noBackground_file.PNG');
+        await uploadDialog.fileIsCancelled('a_png_noBackground_file.PNG');
+
+        await BrowserVisibility.waitUntilElementHasText(uploadDialog.title, 'Upload canceled');
```

Comment:
This line should be part of UploadDialog PO

Reply:
not sure I got this test is anyway excluded for now needs more refactor